### ISSUE TITLE
The table is too wide, reduce the number of columns

### ISSUE THREAD

**Issue by claudepache** [compat-table/compat-table#649]
The following pairs of columns display the same results:
- "SF 6.1, SF 7" and "iOS 7";
- "SF 7.1, SF 8" and "iOS 8";
- "SF 9.0" and (not yet included) "iOS 9".

In order to reduce the width of the table, it may be a good idea to fuse these pairs of columns.

In that regard, having two distinct groups "Desktop browsers" and "Mobile" might be not very relevant.

**Comment by kangax** [comment]
I'd rather keep implementations separated by type. I feel like it's more convenient to have separate mobile section — that way you can quickly find mobile browser, you can also visually compare mobile browsers among each (percentage wise) and so on.

If a width of a table is a problem, perhaps we can think of other ways to improve UI? Make these sections expandable and collapsed by default? Or something else?

**Comment by claudepache** [comment]
Yes, the main problem is the width of the table (a wide screen is barely sufficient), and that was an obvious solution for the issue.

Making the four types hide-able is certainly a better idea. Maybe with checkboxes: "Show Compiler/polyfill", etc.?

**Comment by kangax** [comment]
This might be a controversial solution but how about tabs? Checkboxes seem like a poor man visibility control mechanism. We can have good old tabs instead, show desktop browsers by default, and have mobile, server, compilers in their own tabs. For people that want birds view we can possibly still support fully expanded view (as an option).

**Comment by claudepache** [comment]
Or something like this?

![hide](https://cloud.githubusercontent.com/assets/3309637/10344787/87ffa50e-6d25-11e5-9e1e-cb3556bf833b.png)

**Comment by kangax** [comment]
But then we're still left with these awkward grey empty columns that take some space, no?

**Comment by caitp** [comment]
How about some option to arrange the order the columns are presented in? I'm sure there are libraries for accomplishing that somewhere out there.

For browsers further towards the right, it's very difficult to read, because the description of each row is hidden. Being able to slide a column over, to be closer to the row descriptions, would help a good amount

**Comment by kangax** [comment]
@caitp It still feels like a temporary hack that works around current problems. I think tabs is still the best long-term solution — the entire thing will be 4x less and this problem (I think) will mostly disappear.

**Comment by kangax** [comment]
@caitp I do agree that reading columns on the very right is just a horrific experience at the moment! I'll try to get to this as soon as I get a chance.
